### PR TITLE
Preliminary utilities for Xorg update

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/x11/compat-links/compat-links.p5m [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=description value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+dir group=bin mode=0755 owner=root path=usr/X11
+link path=usr/X11R6 target=X11
+link path=usr/openwin target=X11
```
Comment (pyhalov) [inline components/x11/compat-links/compat-links.p5m:23]: our current package has much more links... Is it OK?

**Comment by pyhalov** [comment]
Shouldn't we also relax components/meta-packages/X-incorporation/X-incorporation.p5m ?

File: components/x11/compat-links/compat-links.p5m (hunk 1 above)
Comment (alarcher) [inline components/x11/compat-links/compat-links.p5m:23]: Openwin links, Xserver links are deprecated, other links for fonts were dropped as well and I could not find consumers.
I can re-add them for safety if you like.

Comment (pyhalov) [inline components/x11/compat-links/compat-links.p5m:23]: I'd add links which we currently provide. Applications sometimes still try to use /usr/openwin/include.

**Comment by alarcher** [comment]
Updated.

**Pull request merged**